PEPit/pep.py

```diff
@@ -575,6 +587,7 @@ def _solve_with_wrapper(self, wrapper, verbose=1, return_primal_or_dual="dual",
         # leading to different dual values. The ones we store here provide the proof of the obtained guarantee.
         self.residual = wrapper.assign_dual_values()
         G_value, F_value = wrapper.get_primal_variables()
+        nb_eigenvalues = G_value.shape[0]
```

Comment:
I do not understand what this variable should contain. Given the attribute it is later stored in, I thought nb_eigenvalues should contain the rank of G_value (excluding the null eigenvalues). Did I miss something?